build/build.go

```diff
@@ -769,6 +777,9 @@ func BuildWithResultHandler(ctx context.Context, nodes []builder.Node, opt map[s
 		}
 	}
 
+	// print build details links
+	logrus.Printf(desktop.BuildDetailsOutput(buildDetailsOpts))
```

Comment:
As discussed, we should probably output to progress instead of logrus.

Reply:
Progress output should be in beginning (and needs more testing with bake). After build this could be a "build summary" section. I think logrus is fine for now and we can turn it into proper summary in next iteration.

Reply:
I think it is better if this logging is in `cmd` packages.